src/lifecycle/application.rs

```diff
@@ -12,7 +12,6 @@ pub struct Application<T: State> {
     pub state: T,
     pub window: Window,
     pub event_buffer: Vec<Event>,
-    accumulator: f64,
```

Comment:
This was indeed something I was thinking about but the sheer fact, that the `update()` method was called inside a while loop was just so bizarre.
In my use case for example, it cause the drawing to lag behind, even though there was enough time to draw. I think my settings were: `max_ticks = 10000`, `tick_rate = 1/60.`

Even though the value is called "max_ticks" quicksilver ALWAYS called `update()` 10000 times instead of aborting when it was time to draw again. This caused massive lags and made my application useless in combination with quicksilver. Thus this PR.